code/game/machinery/computer/cloning.dm

```diff
@@ -6,19 +6,25 @@
 	icon_screen = "dna"
 	icon_keyboard = "med_key"
 	circuit = /obj/item/circuitboard/computer/cloning
+	req_access = list(ACCESS_HEADS) //ONLY USED FOR RECORD DELETION RIGHT NOW.
 	var/obj/machinery/dna_scannernew/scanner = null //Linked scanner. For scanning.
 	var/list/pods //Linked cloning pods
 	var/temp = "Inactive"
 	var/scantemp_ckey
 	var/scantemp = "Ready to Scan"
+	var/menu = 1 //Which menu screen to display
+	var/list/records = list()
 	var/datum/data/record/active_record = null
+	var/obj/item/disk/data/diskette = null //Mostly so the geneticist can steal everything.
 	var/loading = 0 // Nice loading text
+	var/autoprocess = 0

```

Comment:
This looks like a boo from quick read

```suggestion
	var/autoprocess = FALSE
```

Reply:
> a boo
Do you mean you don't like it? Bit confused